spec.md

```diff
@@ -1377,7 +1403,9 @@ node mynode = mux(pred, a, b)
 
 ## Conditionals
```

Comment:
Nit: Add blurb between a header and its first sub-header.  Here, introducing the two types of conditionals seems like a good idea.